xbmc/platform/win32/filesystem/Win32Directory.cpp

```diff
@@ -46,8 +46,6 @@ CWin32Directory::~CWin32Directory(void)
 
 bool CWin32Directory::GetDirectory(const CURL& url, CFileItemList &items)
 {
-  items.Clear();
```

Comment:
Looking at the commit that introduced this in the first place it shouldn't be needed. However I am not sure it won't bring regressions. I'd say to remove it and see what screams later :)
If any, it will for sure be detected soon